sledilnik_mobile_app/lib/main.dart

```diff
@@ -99,7 +99,7 @@ class _MyHomePageState extends State<MyHomePage> {
       ),
       floatingActionButton: FloatingActionButton(
         onPressed: () => null,
-        tooltip: 'Increment',
+        tooltip: 'Pojdo na graf',
```

Comment:
Pojdi :)

Reply:
Oh! Fixed :).
This seemed like a good case for a git amend + force push to avoid a historically insignificant single-char commit to take up space.

Reply:
Same can be achieved by sqash-merging this PR
![image](https://user-images.githubusercontent.com/319826/97841252-b71aaf80-1ce5-11eb-841c-b404770b0854.png)
Ok, when a PR is dealing with a single, single author issue and there are several fixes. PR #11 is another candidate for such squash merge.